controllers/history.controller.js

```diff
@@ -0,0 +1,88 @@
+const History = require("../models/history-model");
+const User = require("../models/user-model");
+const { concat,remove }=require("lodash");
+
+const getHistory=async(req,res)=>{
+  const history=await History.find({});
+  res.json({success:true,history});
+}
+
+const findUserHistory = async (req, res, next, userId) => {
+  try {
+    let user = await User.findOne({ _id: userId });
+    if (!user) {
+      res.status(404).json({
+        success: false,
+        message: "Invalid user! Kindly register to continue",
```

Comment:
I am not sure....but I think a return statement is needed here. Let me know if I am missing anything!

Reply:
Here I thought to throw the custom error. Yeah we can do as you told